cirq-core/cirq/protocols/json_serialization.py

```diff
@@ -209,6 +224,21 @@ def wrap(cls):
 # pylint: enable=redefined-builtin
 
 
+def dataclass_json_dict(obj: Any, namespace: str = None) -> Dict[str, Any]:
+    """Return a dictionary suitable for _json_dict_ from a dataclass.
+
+    Dataclasses keep track of their relevant fields, so we can automatically generate these.
+
+    Dataclasses are implemented with somewhat complex metaprogramming, and tooling (PyCharm, mypy)
```

Comment:
This paragraph is largely duplicated in the module docstring above. I'd suggest removing it here and relying on the module docstring.

Reply:
what do you mean module docstring? do you mean the docstring for json_serializable_dataclass? I think it's relevant to both functions. Especially if we remove json_serializable_dataclass

Reply:
Ah sorry, I thought other occurence of this text was in the `json_serialization.py` module docstring. Agree it's fine to have on `json_serializable_dataclass`.